Tests/Unit/Module/STIG.Checklist.tests.ps1

```diff
@@ -3,16 +3,27 @@
 #endregion
 
 Describe 'New-StigCheckList' {
-
+    # Test parameter validity -OutputPath
```

Comment:
I'll remove it if you really want me to. I start writing tests by first writing comments for each area/component that I'm going to test. It may take more than one test for each area/component. That's why they are there.